src/lib.rs

```diff
@@ -426,14 +430,13 @@ impl Message {
     ///
     /// # Example
     /// ```ignore
-    /// let is_valid: bool = message.verify_eip1271(&signature, "https://cloudflare-eth.com".try_into().unwrap())?;
+    /// let is_valid: bool = message.verify_eip1271(&signature, "https://eth.llamarpc.com".try_into().unwrap())?;
```

Comment:
I am not comfortable with this plug. If you have an issue with Cloudflare would you mind using `http://provider.example` and adding a link to https://ethereumnodes.com/?

Reply:
Cloudflare rate limits are super low and people complain about hitting them a lot. I'll make it generic